deployment/fedora-package-x64/create_tarball.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# shellcheck disable=SC1091
+source ../common.build.sh
+
+WORKDIR="$( pwd )"
+VERSION="$( sed -ne '/^Version:/s/.*  *//p' "${WORKDIR}"/pkg-src/jellyfin.spec )"
+
+package_temporary_dir="${WORKDIR}/pkg-dist-tmp"
+pkg_src_dir="${WORKDIR}/pkg-src"
+
+GNU_TAR=1
+echo "Bundling all sources for RPM build."
+tar \
+--transform "s,^\.,jellyfin-${VERSION}," \
+--exclude='.git*' \
+--exclude='**/.git' \
+--exclude='**/.hg' \
+--exclude='**/.vs' \
+--exclude='**/.vscode' \
+--exclude='deployment' \
+--exclude='**/bin' \
+--exclude='**/obj' \
+--exclude='**/.nuget' \
+--exclude='*.deb' \
+--exclude='*.rpm' \
+-czf "$pkg_src_dir/jellyfin-${VERSION}.tar.gz" \
+-C "../.." ./ || GNU_TAR=0
+
+if [ $GNU_TAR -eq 0 ]; then
+    echo "The installed tar binary did not support --transform. Using workaround."
+    mkdir -p "${package_temporary_dir}/jellyfin"{,-"${VERSION}"}
```

Comment:
a typo?
```suggestion
    mkdir -p "${package_temporary_dir}/jellyfin-${VERSION}"
```

Reply:
No.  Not a typo.  A bugfix.  Both `${package_temporary_dir}/jellyfin` and `${package_temporary_dir}/jellyfin-${VERSION}` need to be exist.

See the first example in this [tutorial on brace expansion in bash](https://www.linuxjournal.com/content/bash-brace-expansion).

The prrof:

```bash
$ echo {aa,bb,cc,dd}
aa bb cc dd
```

Reply:
I know about brace expansion, but I didn't know enough about RPM specs :) Thanks for teaching me!